### ISSUE TITLE
Add deepchem_nightly pip package

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#1955]
I've been doing a bit of research and Travis-CI has pretty decent support for deploying to PyPi:

https://docs.travis-ci.com/user/deployment/pypi/

I propose that we leverage this support to put up a deepchem_nightly package. DeepChem development is proceeding rapidly and there are a lot of utilities that mean developing on nightly is much nicer than 2.3.0. 

If folks are ok with this idea, I'll go ahead and put up a PR

**Comment by nd-02110114** [comment]
Currently, I think we can install deepchem HEAD codes by the following commands.

```
$ pip install git+https://github.com/deepchem/deepchem
```

However, this command is slow because it clones the deepchem repository... (DeepChem repository is a a little big, almost 700MB) So, I seem that this idea is better 👍

**Comment by rbharath** [comment]
#1960 set up the first part of a framework for this, but the twine upload is currently failing as follows:

```
$ twine upload -r pypi dist/*
Uploading distributions to https://upload.pypi.org/legacy/
Uploading deepchem-nightly-2.3.0.tar.gz
100%|██████████| 307k/307k [00:00<00:00, 962kB/s]  
NOTE: Try --verbose to see response content.
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
Invalid value for classifiers. Error: Classifier 'License :: OSI Approved :: MIT' is not a valid classifier.
```

**Comment by rbharath** [comment]
I think to generate release builds, we can use a before-deploy script https://docs.travis-ci.com/user/deployment-v2#running-commands-before-and-after-the-deploy-step and a conditional deploy https://docs.travis-ci.com/user/deployment-v2/conditional alongside Travis-CI support for multiple deployments to generate deepchem builds on tags

**Comment by rbharath** [comment]
Ok, I think deepchem-nightly is now working! https://pypi.org/project/deepchem-nightly/#history

@nd-02110114 Could you try swapping JaxChem install over to deepchem-nightly to see if it works for you?

**Comment by nd-02110114** [comment]
@rbharath 

`deepchem-nightly` works well! I appreciate your working!

**Comment by nd-02110114** [comment]
This issue is fixed by #1986 

DeepChem nightly version is installed by 

```
pip install --pre deepchem
```
